pandas/tests/io/test_parquet.py

```diff
@@ -537,14 +537,24 @@ def test_categorical(self, pa):
             check_round_trip(df, pa, expected=expected)
 
     def test_s3_roundtrip(self, df_compat, s3_resource, pa):
+        s3fs = pytest.importorskip("s3fs")
+        s3 = s3fs.S3FileSystem()
+        kw = dict(filesystem=s3)
+        check_round_trip(
+            df_compat,
+            pa,
+            path="pandas-test/pyarrow.parquet",
+            read_kwargs=kw,
+            write_kwargs=kw,
+        )
+
+    def test_s3_roundtrip_explicit_fs(self, df_compat, s3_resource, pa):
```

Comment:
Naming nitpick: I think it's the test above that has the explicit filesystem passed?